community.md

```diff
@@ -130,6 +130,7 @@ Key:
 
 | Blog/repo name and description                                          | Author       | Site     | Date        |
 |---------------------------------------------------------------------|--------------|----------|-------------|
+| [Serverless Computing: More Time for the Code](https://www.golem.de/news/serverless-computing-mehr-zeit-fuer-den-code-1811-137516.html) | Valentin Höbel | golem.de | 08-Nov-2018 |
```

Comment:
Since people come to this list of articles to read about OpenFaaS, how about pinning this to the link I sent over that mentions OpenFaaS?